db/migrate/20230317132501_create_free_trainings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFreeTrainings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :free_trainings do |t|
```

Comment:
I would call these table exercises.

It is a generic thing, and probably will be just a list that will be associated with other entities.

Does that make sense to you?